cellrank/tools/kernels/_kernel.py

```diff
@@ -155,8 +155,8 @@ def write_to_adata(self, key_added: Optional[str] = None) -> None:
         -------
         None
             Updates the underlying :paramref:`.adata` object with the following:
-                - `.uns[:paramref:`T_{fwd, bwd}` _`:paramref:`key_added`]['T']` - transition matrix
-                - `.uns[:paramref:`T_{fwd, bwd}` _`:paramref:`key_added`]['params']` - parameters used for calculation
+                - `.obsp[`'T_{fwd, bwd}'` _`:paramref:`key_added`]` - transition matrix
+                - `.uns[`'T_{fwd, bwd}'` _`:paramref:`key_added` `'_params'`] - parameters used for calculation
```

Comment:
shouldn't that also be `.obsp`?

Reply:
sorry, my bad, didn't read what you said in the PR. Storing the params in `.uns` is fine.